source/features/edit-comments-faster.tsx

```diff
@@ -7,13 +7,13 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function canEditDiscussion(): boolean {
-	return pageDetect.isDiscussion() && select.exists('[title^="You are a maintainer"], [title^="You are a collaborator"]');
+function canEditComments(): boolean {
+	return select.exists('[aria-label^="You are the owner"], [title^="You are a maintainer"], [title^="You are a collaborator"]');
```

Comment:
Not sure if the maintainer/collaborator checks work

Reply:
Yes, the checks still work.